code_samples/api/public_php_api/src/Command/ViewContentMetaDataCommand.php

```diff
@@ -99,9 +101,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // Relations
         $versionInfo = $this->contentService->loadVersionInfo($contentInfo);
-        $relationCount = $this->contentService->countRelations($versionInfo);
```

Comment:
Maybe this one should be kept.

Reply:
Never displayed, it was only used to load the list. Reader can rediscover it in the PHP API Ref if needed.